spec/models/manageiq/providers/hawkular/middleware_manager/refresh_parser_spec.rb

```diff
@@ -95,4 +92,29 @@
       expect(parser.swap_part(part)).to eq(expected)
     end
   end
+
+  describe 'parse_deployment' do
+    it 'handles data and fetches the enabled metric' do
+      # parse_deployment(server, deployment)
```

Comment:
Is just a reminder of what I'm testing there, I could remove it, below is clear what I'm testing.